catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/util/impl/Permissions.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.util.impl;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+
+public class Permissions {
+
+    /**
+     * Parses a string  array representation of permission attributes into a map
+     * @param permStrings array of permission strings in the format "permName=val1,val2"
+     */
+    public static Map<String, Set<String>> parsePermissionsFromString(String[] permStrings) {
```

Comment:
If you're going to make this a utility, you should probably also support List<String> and String...